viewsb/decoders/standard_descriptors.py

```diff
@@ -378,10 +378,10 @@ def find_specialized_descriptor(cls, data, interface_descriptor, subtype):
             is_device = False
 
         # Search all of our subclasses.
-        for subclass in cls.__subclasses__():
-            matches = subclass.matches_class_specifics(usb_class, subclass, protocol, subtype, is_device)
+        for subcls in cls.__subclasses__():
+            matches = subcls.matches_class_specifics(usb_class, subclass, protocol, subtype, is_device)
             if matches:
-                return subclass
+                return subcls
```

Comment:
Is this actually a bug I'm unaware of? It looks like this is just an inconsequential variable name change.

Reply:
it is indeed a bug. Look at the .matches_class_specifics() parameters.

Reply:
Hopefully https://lgtm.com/projects/g/usb-tools/ViewSB/snapshot/bd64b9054d31b050e2a3eac5c929b9dd89ca13b7/files/viewsb/decoders/standard_descriptors.py#x325cf51d171fd96c:1 makes it clearer what this change is about. I dry-ran this code and agree with the analysis